### ISSUE TITLE
Types of Cancer Research Methods – latest and updated

### ISSUE THREAD

**Issue by emmaosuna145** [emmaosuna145/Emma#3]
**We hear a lot about the need for cancer research funding** and more than likely you have heard about some of the projects currently being funded. These projects are as good as the research done on cancer cells, and it is those that lead to better treatments that make people stand up and cheer. _However, you might not realize there are many different types of research being done_ with the intention of finding cures for cancer.

**When you go into the world of cancer research, you will find that there** are three main areas. The first one is clinical cancer research, which seeks to identify what causes and predisposes patients to getting cancer in the first place. _It is a project that are very specific and yet no one seems to be able to quite define what causes this type of cancer in some people_, but it has been known to occur quite often.

**There are a few methods that clinical [cancer research]**(https://www.clinicalscoop.com/ ) uses. One way is called tissue imaging. T_his method allows researchers to identify the exact tumors and take images of them_, so they can know exactly what parts of the body will be studied.

**Pathogen Based Research methods – cancer research**

**Another form of clinical cancer research is called pathogen**-based research. _Pathogen research involves taking small samples of tissue from different areas of the body_ to use as a basis for research. They know the exact cause of the disease and can then begin working on a cure.

**Radiation therapy research is the other area of cancer research**. This involves the use of radiation to treat various forms of cancer. There are two types of radiation that are used. _The first one is called radiation oncology, which involves the use of X-rays or other forms of radiation to kill cancer cells in certain parts of the body._

**Radiation therapy – cancer treatment**

**The second type of [radiation therapy is called therapeutic](https://www.britannica.com/science/radiation-therapy)** radiation therapy, which is used to treat tumors that are just starting to grow. The difference between the two types of radiation is that in the first type, _the radiation doesn't have a long-term effect on the body. It kills the cancer cells but does not go on to affect the healthy cells._

**When it comes to clinical cancer research involves** the latest studies that are being conducted in the area of oncology. Many times this type of research will help to determine _what type of treatment may work best in treating a certain type of cancer_. Sometimes this type of research will even help to prevent certain types of cancer from happening again.

**Gene therapy cancer research**

For instance, research done on **gene therapy can help to determine if there are any limitations** to how much of a certain gene can be used for the treatment of cancer. _In many cases this type of research can reduce the number of cancer patients_ who have to undergo treatments like radiation. Other types of cancer research are also carried out in the hopes of figuring out a cure for cancer that has already developed.

**Many times these types of cancer research methods are used for testing**. However, sometimes they are used to try and discover a cure for cancer that hasn't developed yet. _This is a very important area of research, as the hope is that if we can get rid of the cancer cells before they develop, we will be able to stop that from happening in the future._

**Clinical cancer research** 

**Another type of clinical cancer research is called [biomarker research](https://biomarkerres.biomedcentral.com/)**. This involves trying to figure out which parts of the body are affected by some chemical that is present in cancer cells. As a result, _it is possible to create a test that will tell doctors if a certain substance is present in a patient's body, and if it is present it can be used as a test for cancer prevention_.

**This type of medical research could be very helpful for cancer research** as well. For instance, if we can figure out which chemicals are present in the blood of cancer patients, it may be possible to _determine what treatments would be most effective. This may be good news for cancer patients and doctors._

**In conclusion, there are plenty of different types of cancer research** methods being applied to many different types of cancers. These are all intended to help find ways to stop cancer from _spreading in the body and help to keep it from coming back once it has started._
